lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -834,6 +834,11 @@ private function parse_next_attribute( $context = 'tag-opener' ) {
 			return true;
 		}
 
+		// Continue parsing if no attribute name. No point in storing something as ''/false.
```

Comment:
can you add a `@TODO: Remove this after fixing PHP version inconsistency with strspn`